pkg/platformconfig/types.go

```diff
@@ -261,3 +262,51 @@ type StreamMonitoringConfig struct {
 	WebapiURL              string `json:"webapiURL,omitempty"`
 	V3ioRequestConcurrency uint   `json:"v3ioRequestConcurrency,omitempty"`
 }
+
+type SensitiveFieldPath string
+
+type SensitiveFieldsConfig struct {
+
+	// CustomSensitiveFields is a list of fields that should be masked in logs and function config
+	CustomSensitiveFields []string `json:"sensitiveFields,omitempty"`
+	SensitiveFieldsRegex  []*regexp.Regexp
+	MaskSensitiveFields   bool
+}
+
+func (sfc *SensitiveFieldsConfig) GetDefaultSensitiveFields() []string {
+	return []string{
+
+		// build
+		"^/Spec/Build/CodeEntryAttributes/password",
+		// volumes
+		"^/Spec/Volumes\\[\\d+\\]/Volume/VolumeSource/FlexVolume/Options/accesskey",
+		// triggers - global
+		"^/Spec/Triggers/.+/Password",
+		"^/Spec/Triggers/.+/Secret",
+		// triggers - specific
+		// - v3io stream
+		"^/Spec/Triggers/.+/Attributes/password",
+		// - kinesis
+		"^/Spec/Triggers/.+/Attributes/accessKeyID",
+		"^/Spec/Triggers/.+/Attributes/secretAccessKey",
+		// - kafka
+		"^/Spec/Triggers/.+/Attributes/caCert",
+		"^/Spec/Triggers/.+/Attributes/AccessKey",
+		"^/Spec/Triggers/.+/Attributes/AccessCertificate",
```

Comment:
are they case sensitive? imo we should be case insensitive here(aka, lowercase() all field paths

Reply:
You're right, added ignore case in regex.